README.md

```diff
@@ -102,25 +102,29 @@ You will need to restart your Jekyll server to see the effects.
 To develop this theme and/or test it locally:
 
 1. Clone this repo
-
 1. Run Jekyll (`jekyll serve`) in the local clone of this repo;
    **or**
-
 1. Create a new Jekyll project, follow the
    [installation](#installation) instructions, then change your
    `Gemfile` to point at the local clone of this repo:
 
     ```ruby
     gem 'uswds-jekyll', :path => '../path/to/uswds-jekyll'
     ```
+### Publish to Rubygems
+1. Update `spec.version = "NUMBER HERE"` in the uswds-jekyll.gemspec file to the version you want to publish
+1. Run `bundle install`
+1. Run `bundle exec rake release`
+1. Add a PR for the update and get it merged
```

Comment:
I'm wondering if this step should be done before `bundle exec rake release`? Presumably you'd want a second pair of eyes on the PR prior to publishing it, but I may not be following the process correctly.

Reply:
@jeremiak ❤️ to get your thoughts here

Reply:
Ya, I think @hursey013 is correct. `rake release` will push to RubyGems so probably best to run the first two steps and _then_ open the PR. Following the merge, then run `rake release`.